### ISSUE TITLE
Move to ebiten?

### ISSUE THREAD

**Issue by drjole** [exp625/gones#2]
@exp625 In your research, did you consider [ebiten](https://ebiten.org/)?

By moving to ebiten (which would not require much effort in the current development state, I guess), I see the following advantages:

- WASM Support (-> gones in the browser 😱)
- No need for CGO on windows

**Comment by exp625** [comment]
WASM and no CGO looks very interesting. I didn't look at the package when I was looking for a 2D library. I played a bit with the libary and it was pretty easy to  move to ebiten. However, I did find two problems that I couldn't fix yet:
- The very nice pixel font we currently use needs to be converted to the format that ebiten uses.
- I have not found an easy way to change the text color for only a portion of the text. The text.Draw only accepts a regular string and a global color.

Maybe you want to take a look at it. See the ``ebiten`` branch
